### DISCUSSION TITLE
Does the latest still need X-API-Key header?

### DISCUSSION THREAD

**Discussion by mzch** [james-stevens/powerdns-webui#49] [Q&A] [answered]
On login screen it's necessary to input API Key and `index.html` sends it to PowerDNS.
It seems to be no longer necessary that X-API-Key header in apache.conf (nor nginx.conf)  is set.
Right?

**Comment by james-stevens** [comment, marked as answer]
Right.

Its better to use an HTTP proxy with per-user authentication and put the API Key in the HTTP server's config - giving out shared keys to users is generally a bad idea. If you are using an HTTP proxy, then you do *not* need to provide the API Key at login.

However, some people prefer to keep their set up very simple and talk directly to PowerDNS API (no HTTP Proxy). If they wish to do this, then they need to b able to provide the API Key when attaching.

**Comment by mzch** [comment]
Ah, I see. Thanks! :)
